packages/common/src/services/oauth/formatSocialProfile.ts

```diff
@@ -109,10 +109,9 @@ export const formatInstagramProfile = async (
     instagramProfile.full_name.length > MAX_DISPLAY_NAME_LENGTH
   ) {
     requiresUserReview = true
-    instagramProfile.full_name = instagramProfile.full_name!.slice(
-      0,
-      MAX_DISPLAY_NAME_LENGTH
-    )
+    if (instagramProfile.full_name) {
+      instagramProfile.full_name.slice(0, MAX_DISPLAY_NAME_LENGTH)
```

Comment:
Slice doesn't change orignal array right?

Reply:
oh yeah lol

Reply:
i forgot to put the assignment back when i moved `instagramProfile.full_name` it to the if statement 💀 thx for catching